### DISCUSSION TITLE
no outbound networking on first boot as Proxmox VM

### DISCUSSION THREAD

**Discussion by nrvale0** [SpiralLinux/SpiralLinux-project#504] [Q&A] [answered]
Howdy, folks.

Very strange behavior here. I can ssh and ping **to** my Spiral box at first boot after ISO install as a Proxmox VM but I have no outbound networking. No ping, no DNS resolution. But I can see ARP addresses on the LAN.

No firewall enabled within Proxmox and it doesn't look like Spiral has ufw stuff enabled by default. Many other VMs and containers on this same Proxmox host and they are all just fine with networking.

The VM gets a DHCP address on boot but if I stop the interface and restart I then do not get a DHCP address.

Sort of at a loss. 

Thoughts?

**Comment by nrvale0** [comment]
It probably doesn't matter but this is Cinnamon variant.

Also, many many other Debian-based VMs on this Proxmox host + network.

**Comment by nrvale0** [comment, marked as answer]
Solved: very odd but DHCP server handed out address twice. Not anything Spiral can do about THAT!. :D

**Reply by geckolinux** [reply]
Glad you fixed it! Thanks for posting the resolution.
